### ISSUE TITLE
Add high contrast mode with `prefers-contrast`

### ISSUE THREAD

**Issue by tempertemper** [tempertemper/tempertemper.net#594]
A high contrast mode should be added for users that have their operating system's 'increase contrast' setting on. The high contrast version should adhere to AAA for colour contrast.

Some resources:

- [Specification](https://drafts.csswg.org/mediaqueries-5/#prefers-contrast)
- [Browser support](https://caniuse.com/mdn-css_at-rules_media_prefers-contrast)
- [CSS Tricks news item](https://css-tricks.com/platform-news-prefers-contrast-mathml-is-and-css-background-initial-values/#the-prefers-contrast-more-media-query-is-supported-in-safari-preview)
- [MDN docs](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-contrast)

This is the sort of thing to add:

```
@media (prefers-contrast: more) {
  background-color: black;
}
```

**Comment by tempertemper** [comment]
Tried adding this in March 2021 as it's supported by the newest Safari Technology Preview, but the 'Increase contrast' setting in System Preferences doesn't trigger the expected behaviour. Going to wait a while until it all gets fixed.
